CHANGELOG.md

```diff
@@ -3,6 +3,50 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.7
+
+_Nov 4, 2021_
+
+- 👁 Move virtualization logic to hook (#3079) @m4theushw
+- 🌍 Add hebrew localization (he-IL) (#3028) @ColdAtNight
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.0.0-beta.7` / `@mui/x-data-grid-pro@v5.0.0-beta.7`
+
+#### Breaking changes
```

Comment:
You have to go on each merged PR and check if there're breaking changes and copy them here. For instance, #3002 has breaking changes. I'm not gonna copy here because the diff doesn't work correctly with code block included in the suggestion.

<img width="400" src="https://user-images.githubusercontent.com/42154031/140308335-0165b910-f596-4c10-a1b3-aa6181085b42.png" alt="image" style="max-width: 100%;">